extensions/plans.yml

```diff
@@ -126,6 +126,9 @@ opensearch:
     instance: m6g.2xlarge.search
 
 # RDS/Postgres Instances
+# Todo: align these with the instance types from https://aws.amazon.com/rds/instance-types/
+  # WILL BE BACKWARDS BREAKING
+  # ALL THE EXISTING PLANS MUST STILL EXIST EVEN IF UNDER DIFFERENT NAMES, OTHERWISE WE RISK DATABASES BEING RE-PROVISIONED
```

Comment:
We will need to shout loud about this change. Tagging @yusufsheiqh and @acodeninja to make sure they know it's coming.

I don't think we need to change https://github.com/uktrade/platform-tools/blob/main/dbt_platform_helper/addon-plans.yml. Hoping that can stay as is so we don't introduce risk for any services we have still using the fully AWS Copilot platform tooling.

Reply:
@yusufsheiqh @acodeninja Will any of the changes here be an issue for you?

Reply:
This isn't an issue and have we confirmed if databases will need to be provisioned again for the upgraded storage types?

Reply:
From https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#Concepts.Storage

Sentences like these tell me that it's a non destructive action...

"When you modify a General Purpose SSD or Provisioned IOPS SSD volume, it goes through a sequence of states. While the volume is in the optimizing state, your volume performance is in between the source and target configuration specifications."

"Amazon RDS provisions new volumes and transparently moves the data from the old volume to the new volumes."

This also ties up with @tony-griffin's empirical results.